src/components/date/__internal__/date-picker/date-picker.component.tsx

```diff
@@ -1,37 +1,44 @@
-import React, { useCallback, useEffect, useMemo, useRef } from "react";
-import DayPicker, {
+import { flip, offset } from "@floating-ui/dom";
+import React, {
+  useCallback,
+  KeyboardEvent,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from "react";
+import {
+  DayPicker,
   DayPickerProps,
-  DayModifiers,
-  Modifier,
-  LocaleUtils,
+  defaultLocale,
+  Modifiers,
 } from "react-day-picker";
-import { flip, offset } from "@floating-ui/dom";
 
-import { getDisabledDays } from "../utils";
-import Popover from "../../../../__internal__/popover";
 import useLocale from "../../../../hooks/__internal__/useLocale";
+import Popover from "../../../../__internal__/popover";
 import Navbar from "../navbar";
 import Weekday from "../weekday";
-import StyledDayPicker from "./day-picker.style";
-import Events from "../../../../__internal__/utils/helpers/events";
+import { getDisabledDays } from "../utils";
 import { defaultFocusableSelectors } from "../../../../__internal__/focus-trap/focus-trap-utils";
+import Events from "../../../../__internal__/utils/helpers/events";
+
+import StyledDayPicker from "./day-picker.style";
+
+import "react-day-picker/style.css";
```

Comment:
question (non-blocking): do we get anything that we need from this? I tried removing it and couldn't see any regressions but I could be missing something. Happy for it to be non-blocking as it's not a big deal including it and the jest config update etc